### ISSUE TITLE
Version numbers

### ISSUE THREAD

**Issue by rupert** [renalreg/radar#165]
We need to be able to release new staging versions without updating the main version number.

http://fedoraproject.org/wiki/Packaging%3aNamingGuidelines#NonNumericRelease

The release number could be generated using something like:

```
$ git rev-list --count HEAD
```

Including the commit hash will be useful too:

http://fedoraproject.org/wiki/Packaging%3aNamingGuidelines#Snapshot_packages

**Comment by rupert** [comment]
Doesn't work well for shallow clones.

```
$ git clone --depth=50 git@github.com:renalreg/radar.git shallow
$ cd shallow
$ git rev-list --count HEAD
50
```

**Comment by rupert** [comment]
An alternative is to use the last commit date:

```
$ git log --format="%h" -n 1
915d344
$ git log -1 --format="%cd" --date=iso-strict
2015-11-06T18:12:56-08:00
```

```python
>>> from delorean import parse
>>> x = parse("2015-11-06T18:12:56-08:00")
>>> x.datetime.strftime("%Y%m%d%H%M%S")
'20151107021256'
```

Convert to UTC and set the release number to `0.20151107021256` and the `{%checkout}`  to `20151107git915d344`.

**Comment by George-D-S** [comment]
There's also the thing PV2 does where it refers to it by the build date.

**Comment by rupert** [comment]
Fixed in 6505cb9a47cb01212fb6db6c8b664b8e570e9a4b
